README.md

```diff
@@ -27,12 +27,79 @@ The grape-swagger gem provides an autogenerated documentation for your [Grape](h
 
 These screenshot is based on the [Hussars](https://github.com/LeFnord/hussars) sample app.
 
+## Model Parsers
+
+Since 0.21.0, `Grape::Entity` is not a part of grape-swagger, you need to add `grape-swagger-entity` manually to your Gemfile.
+Also added support for [representable](https://github.com/apotonick/representable) via `grape-swagger-representable`.
+
+```ruby
+# For Grape::Entity ( https://github.com/ruby-grape/grape-entity )
+gem 'grape-swagger-entity'
+# For representable ( https://github.com/apotonick/representable )
+gem 'grape-swagger-representable'
+```
+
+##### Custom model parser interface
+
+You can create your own model parser, for example for [roar](https://github.com/apotonick/roar)
+
+```rb
+module GrapeSwagger
+  module Roar
+    class Parser
+      attr_reader :model
+      attr_reader :endpoint
+
+      def initialize(model, endpoint)
+        @model = model
+        @endpoint = endpoint
+      end
+
+      def call
+        # Parse your model and return hash with model schema for swagger
+      end
+    end
+  end
+end
+```
+
+Then you should register your custom parser:
+
+```rb
+GrapeSwagger.model_parsers.register(GrapeSwagger::Roar::Parser, Roar::Decorator)
+```
+
+To controll model parsers sequence, you can insert your Parser before or after another parser
```

Comment:
Spelling: control with one l.

Make parser lowercase, add a period in the end.